File: 01_gpio_digital/led_blink.py
import RPi.GPIO as GPIO
import time

LED_red = 4
GPIO.setmode(GPIO.BCM)   #GPIO.BCM or GPIO.BOARD
GPIO.setup(LED_red, GPIO.OUT)
GPIO.output(LED_red, GPIO.HIGH)  #1
print("led on")
time.sleep(2)
GPIO.output(LED_red, GPIO.LOW)#0
print("led off")

LED_yello = 17
GPIO.setup(LED_yello, GPIO.OUT)
GPIO.output(LED_yello, GPIO.HIGH)  #1
print("led on")
time.sleep(2)
GPIO.output(LED_yello, GPIO.LOW)#0
print("led off")

LED_green = 26
GPIO.setup(LED_green, GPIO.OUT)
GPIO.output(LED_green, GPIO.HIGH)  #1
print("led on")
time.sleep(2)
GPIO.output(LED_green, GPIO.LOW)#0
print("led off")

GPIO.cleanup()  #GPIO핀상태 초기화w

File: 01_gpio_digital/led_blink.c
#include <wiringPi.h>
#define LED_red 4
#define LED_yello 17
#define LED_green 26


int main (void)
{

  wiringPiSetupGpio () ;
  pinMode (LED_red, OUTPUT) ;
  
    digitalWrite (LED_red, HIGH) ; delay (1000) ;
    digitalWrite (LED_red,  LOW) ; delay (1000) ;
  pinMode (LED_yello, OUTPUT) ;
 
  
    digitalWrite (LED_yello, HIGH) ; delay (1000) ;
    digitalWrite (LED_yello,  LOW) ; delay (1000) ;
 
  pinMode (LED_green, OUTPUT) ;
 
 
    digitalWrite (LED_green, HIGH) ; delay (1000) ;
    digitalWrite (LED_green,  LOW) ; delay (1000) ;
 
  return 0 ;
}